libs/etterlatte-ktor/src/main/kotlin/feilhaandtering/StatusPagesKonfigurasjon.kt

```diff
@@ -173,10 +173,11 @@ class StatusPagesKonfigurasjon(
     private suspend fun hentRequestobjekt(call: ApplicationCall): String {
         val requestobjekt =
             try {
-                call.receiveText()
+                call.receive<String>()
```

Comment:
Alternativt `call.receive<T>` om requestObjekt er av type `application/json`

Reply:
Ja, her vil nok kompilatoren skjønne at det uansett skal vera string, sia det er returobjektet frå metoden, så kanskje unødvendig å spesifisere her. Men poenget er å få objektet som rein tekst, for å sjå kva det er som gjer at den vanlege deserialiseringa vår (som vel stort sett er nettopp `call.receive<T>` feilar. Som oftast er det vel eit påkravd felt som manglar verdi, men det er ikkje alltid lett å sjå utan å ha sjølve objektet)